pkgs/development/python-modules/scipy/default.nix

```diff
@@ -95,22 +102,23 @@ buildPythonPackage {
       --replace-fail "pybind11>=2.12.0,<2.13.0" "pybind11>=2.12.0" \
   '';
 
-  build-system = [
-    cython
-    gfortran
-    meson-python
-    nukeReferences
-    pythran
-    pkg-config
-    wheel
-    setuptools
-  ] ++ lib.optionals stdenv.isDarwin [
-    # Minimal version required according to:
-    # https://github.com/scipy/scipy/blob/v1.14.0/scipy/meson.build#L185-L188
-    (xcbuild.override {
-      sdkVer = "13.3";
-    })
-  ];
+  build-system =
+    [
+      cython
+      gfortran
+      meson-python
+      nukeReferences
+      pythran
+      pkg-config
+      setuptools
+    ]
+    ++ lib.optionals stdenv.isDarwin [
+      # Minimal version required according to:
+      # https://github.com/scipy/scipy/blob/v1.14.0/scipy/meson.build#L185-L188
+      (xcbuild.override {
+        sdkVer = "13.3";
+      })
+    ];
```

Comment:
I hate this formatting style :cry:...

Reply:
Well, I also sometimes don't, but it is the new official formatting from now on.
We might better just get used to it.

Reply:
> Well, I also sometimes don't, but it is the new official formatting from now on. We might better just get used to it.

Now I noticed that it CI didn't complain about the formatting because somehow. I tend to want to still keep it as is - and CI won't bug me in the future. I'm also a bit optimistic that I will have time to convince the formatting team that the current formatting for this part is OK.